moveit_simple_msgs/srv/TrajectoryTrainer.srv

```diff
@@ -0,0 +1,19 @@
+uint8 NEW_WP=0				# Create a new waypoint at current location 			[required: waypoint name, optional: trajectory name]
+uint8 DEL_WP=1				# Delete existing waypoint(s) from config 				[required: waypoint name(s)]
+uint8 GET_WP=2				# Request information on existing waypoint(s) 			[required: waypoint name(s)]
```

Comment:
This is the only `op code` used in `moveit_simple`.  What are the others used for?

Reply:
The others are used in creation and maintenance of trajectories. Use concept is that a robot can be manipulated (e.g. with the pendant) and its position can be stored through a service call. Trajectories are not in use in moveit_simple yet, but could be used in the future to retrieve sparse trajectories rather than adding points individually.

Reply:
From a moveit_simple perspective, this is not utilized.  My general feeling is that this service is trying to be too much.  We can have a full featured set of services/op codes for nodes that edit the trajectory, but for those that simply read it, why not create a separate service.